Packs/FeedOffice365/Integrations/FeedOffice365/FeedOffice365.py

```diff
@@ -159,22 +155,34 @@ def fetch_indicators(client: Client, indicator_type_lower: str, limit: int = -1)
             values = item.get(indicator_type_lower)
         if values:
             for value in values:
-                type_ = client.check_indicator_type(value)
+                type_ = Client.check_indicator_type(value)
                 raw_data = {
                     'value': value,
                     'type': type_,
                 }
                 for key, val in item.items():
                     if key not in ['ips', 'urls']:
                         raw_data.update({key: val})
+
+                indicator_mapping_fields = {
+                    "port": argToList(item.get('tcpPorts', '')),
+                    "service": item.get('serviceArea', '')
+                }
+
+                if item.get('expressRoute', None):
```

Comment:
```suggestion
                if item.get('expressRoute'):
```
The default value of `.get()` is `None` - no need to explicitly set it to `None`. The same goes for wherever else you explicitly set the default value to `None`.